src/Filter.js

```diff
@@ -119,9 +88,12 @@ function Filter({
     <div className="dq-main-container">
       <div className="dq-heading">
         <div className="dq-heading-main"></div>
-        <div data-testid="agg" className="dq-heading-total-rows">
-          Total rows: {total && total.toLocaleString()}
-        </div>
+        <Totalrows
```

Comment:
@cotts `Totalrows` is undefined and it's causing tests to fail. Also, variable name should be in camelcase.